lib/elixir/lib/kernel.ex

```diff
@@ -3271,7 +3271,12 @@ defmodule Kernel do
       if Map.has_key?(fields, :message) do
         @spec message(Exception.t) :: String.t
         def message(exception) do
-          exception.message
+          cond do
+            is_nil(exception.message) ->
```

Comment:
yes, it makes sense, but still wil be the original error raised,
since the exception is caught by `Exception.message/1`
https://github.com/elixir-lang/elixir/blob/master/lib/elixir/lib/exception.ex#L48-L54